src/Illuminate/Bus/Dispatcher.php

```diff
@@ -109,9 +109,7 @@ public function dispatchNow($command, $handler = null)
     {
         $uses = class_uses_recursive($command);
 
-        if (in_array(InteractsWithQueue::class, $uses) &&
-            in_array(Queueable::class, $uses) &&
-            ! $command->job) {
+        if (isset($uses[InteractsWithQueue::class], $uses[Queueable::class]) && ! $command->job) {
```

Comment:
@GrahamCampbell as @ralphjsmit points out, the result of `class_uses_recursive()` is a key-value pair with the trait name for _both_, therefore you can use `isset()` to check for the usage of a trait (`O(1)`) rather than traverse the array till you find the value using `in_array()`. (`O(n)`)

Reply:
This should be fine and we use `isset()` as well for https://github.com/laravel/framework/blob/eb625fa6aa083dbae74b4f2cc7dc6dafcce5ff07/src/Illuminate/Foundation/Testing/Concerns/InteractsWithTestCaseLifecycle.php#L196-L220

Reply:
@crynobone the `array_flip()` in that example should probably be removed too… (unrelated to this PR)